packages/runtime/container-runtime/src/gcSweepReadyUsageDetection.ts

```diff
@@ -0,0 +1,144 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ITelemetryProperties } from "@fluidframework/common-definitions";
+import { ICriticalContainerError } from "@fluidframework/container-definitions";
+import {
+    IConfigProvider,
+    IFluidErrorBase,
+    LoggingError,
+    MonitoringContext,
+} from "@fluidframework/telemetry-utils";
+import { oneDayMs } from "./garbageCollection";
+
+/** Feature Gate Key -
+ *  How many days between closing the container from this error (avoids locking user out of their file altogether)
+ */
+export const blackoutPeriodDaysKey = "Fluid.GarbageCollection.Dogfood.SweepReadyUsageDetection.BlackoutPeriodDays";
```

Comment:
SkipClosureForDays? Just to make it very explicit what it means? Blackout period kind of sounds the opposite right, like when airlines have blackout days, you don't get any discounts, etc. :D

Reply:
haha ok yeah sure, thanks.